tests/test_cursor.py

```diff
@@ -13,6 +14,11 @@
 from .utils import gc_collect
 
 
+def gc_collect():
```

Comment:
Note that now there is a definition of `gc_collect()` in `.utils`, see line above